src/FluentValidation/Internal/MessageFormatter.cs

```diff
@@ -93,21 +92,26 @@ public MessageFormatter AppendPropertyValue(object value)
 		/// </summary>
 		public Dictionary<string, object> PlaceholderValues => _placeholderValues;
 
-		protected virtual string ReplacePlaceholderWithValue(string template, string key, object value) {
-			string placeholder =  GetPlaceholder(key);
-			return template.Replace(placeholder, value?.ToString());
-		}
+		// TODO: Add alignment component

```

Comment:
What does "Add alignment component" mean?

Reply:
padding. https://docs.microsoft.com/en-us/dotnet/standard/base-types/composite-formatting

Reply:
Let's remove the todo comment. Can revisit later if needbe.